Runtime/Scripts/CarrierInfoIntegration.cs

```diff
@@ -358,6 +356,13 @@ public string GetMccMnc()
 
     public ulong GetCellID()
     {
+      /*
+       * The following code is commented out to prevent Android JNI blacklist crashes.
+       * As of Android API 28, CellInfo interfaces and class reflection through JNI are not allowed.
```

Comment:
instead of leaving it commented, can we detect what the android version is with something like this : 
```
   using (var version = new AndroidJavaClass("android.os.Build$VERSION")) {
     return version.GetStatic<int>("SDK_INT");
   }
}
```
from : https://answers.unity.com/questions/976506/how-to-get-andriod-api-level.html

Reply:
Good Point, @franklin-huang-mobiledgex  since you worked in this bug, Do u think we should detect the Android SDK Version, then get Cell ID if the SDK Version is below 28.

 *btw we don't need versions below 24 since this the minimum we support.

Reply:
If anybody starts using CellID with Android SDK version 24-27, we can do that. The code change is simple, as we already grab the SDK version, because it takes us down slightly different code paths. But for now it's all consistent when it is commented out. Either way is not a big deal; Android JNI is easily crashable, so leaving it commented out is less risky

Reply:
Ok